styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,53 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-imageWrapper {
+  display: none;
+}
+
+.leadership-linkWrapper {
+  margin-top: 0;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
```

Comment:
Try this, and then swap out your `center-block` for `leadership-individualCenter`
```
.leadership-individualCenter {
  float: none;
  margin-left: auto;
  margin-right: auto;
}
```